packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -51,9 +52,13 @@ export const SubscriptionItem = ({
     return (
       <DialogMessage className="dialog-error" onDismiss={locationReload}>
         <h4 data-testid="error-fxa-missing-subscription">
-          Problem loading subscriptions
+          <Localized id="sub-item-missing">
```

Comment:
This one `<Localized />` component is inside the `<h4>` - but all the other changes have `<Localized />` wrapped around markup. Probably just a nit, but should the `<h4>` move inside like other spots?

Reply:
It should, thanks for catching that